package.json

```diff
@@ -2,6 +2,7 @@
   "name": "vyper-remix-plugin",
   "version": "0.1.0",
   "private": true,
+  "homepage": "./"
```

Comment:
why it's not `./build/index.html` ? I'm not sure I understand what homepage is doing

Reply:
Gd